channel/persistence/test/channel_internal_test.go

```diff
@@ -15,22 +15,33 @@
 package test
 
 import (
-	"math/rand"
 	"testing"
 
+	"github.com/stretchr/testify/require"
 	"perun.network/go-perun/wallet"
-	"polycry.pt/poly-go/test"
+	"perun.network/go-perun/wallet/test"
+	pkgtest "polycry.pt/poly-go/test"
 )
 
 func TestRequireEqualSigsTX(t *testing.T) {
-	prng := test.Prng(t)
+	prng := pkgtest.Prng(t)
+	acc := test.NewRandomAccount(prng)
+	data1 := make([]byte, prng.Int63n(50))
+	prng.Read(data1)
+	data2 := make([]byte, prng.Int63n(50))
```

Comment:
Please use a const instead of `50`. The const name should describe the role of this value. (Wondering why the linter does not catch this.)